src/activities/generate-version-json.ts

```diff
@@ -64,8 +42,8 @@ export const generateVersionJson: GenerateVersionJson = async ({ importContentRe
     versionDoi: version.doi,
     article: articleStruct,
     preprintDoi: version.preprint.doi,
-    preprintUrl: version.preprint.content,
-    preprintPosted: version.preprint.publishedDate,
+    preprintUrl: version.preprint.content ?? '',
+    preprintPosted: version.preprint.publishedDate ?? new Date(),
     sentForReview: version.sentForReviewDate,
     peerReview: importContentResult.reviewData ?? version.peerReview,
```

Comment:
I know this isn't a change in this PR (and is probably me), but along the same lines as above, this assumption that we should publish without review content might be a bad one.